src/components/Common.js

```diff
@@ -2,6 +2,12 @@ import styled, { css } from 'styled-components'
 import { text } from './Typography'
 import { P } from './Typography';
 
+const screenBreakpoints = [576, 768, 992, 1200]
+
+export const mediaQueries = screenBreakpoints.map(
```

Comment:
Thoughts on making this a map and labelling them like `xs` `mobile` `tablet` `desktop` or similar?

Reply:
gottem